nautobot_golden_config/management/commands/utils.py

```diff
@@ -0,0 +1,96 @@
+"""Util functions that are leveraged by the managed commands."""
+# pylint: disable=too-many-branches,bad-option-value
+import time
+import uuid
+
+from django.contrib.contenttypes.models import ContentType
+from django.utils import timezone
+from django.test.client import RequestFactory
+
+from nautobot.extras.choices import JobResultStatusChoices
+from nautobot.extras.models import JobResult
+from nautobot.extras.jobs import run_job
+from nautobot.dcim.models import Device
+from nautobot.users.models import User
+
+
+# Largely based on nautobot core run_job command
```

Comment:
Might be good to add details on how this differs from `runjob`, what necessitated copying it here instead of just using `runjob` itself.